pkg/formatter/formatter_internal_test.go

```diff
@@ -37,3 +39,42 @@ func TestStringOrPlaceholder(t *testing.T) {
 	assert.Equal(t, "placeholder", stringOrPlaceholder("", "placeholder"))
 	assert.Equal(t, "foo", stringOrPlaceholder("foo", "placeholder"))
 }
+
+func TestSortCookbookRecords(t *testing.T) {
+  records := []*reporting.CookbookRecord{
```

Comment:
I would suggest to improve the test data. at least 5-10 registries with multi-version cookbooks.